content/departments/engineering/teams/devops/opsgenie-routing.md

```diff
@@ -0,0 +1,31 @@
+# Opsgenie Routing
+
+Our team has decided to split up on call based on the people on the east and west side of the Atlantic ocean. Here are the steps and screenshot of what we did.
+
+## Step 1
+
+Create separate schedules for each team. We have search-core-support-east and search-core-support-west.
```

Comment:
how does this work with slackgenie? it looks like slackgenie only allows a 1:1 mapping for slack handle to schedule - might want to note the compatibility here, since slackgenie is widely used

Reply:
I modified Slackgenie to accept multiple schedules. I'll update the docs on that part too.

Reply:
Wow nice! 😁 😁